### PR TITLE
Clarify that Circle and Buildkite are unsupported

### PR DESCRIPTION
**Pull request by nbrahms**

### CODE REVIEW

File: docs/integrations.md [hunk 1]
```diff
@@ -96,7 +69,46 @@ semgrep:
 
 </p>
 </details>
-</br>
+
+### Standalone providers
```
Comment (pabloest) [inline docs/integrations.md:73]:
```suggestion
## Standalone providers
```

File: docs/integrations.md [hunk 2]
```diff
@@ -96,7 +69,46 @@ semgrep:
 
 </p>
 </details>
-</br>
+
```
Comment (pabloest) [inline docs/integrations.md:72]:
```suggestion
<br />
```

File: docs/integrations.md [hunk 3]
```diff
@@ -96,7 +69,46 @@ semgrep:
...
+        docker:
+            - image: returntocorp/semgrep-agent:v1
+        steps:
+            - checkout
+            - run: python -m semgrep_agent --config p/<ruleset>
+```
+
+</p>
+</details>
+
```
Comment (pabloest) [inline docs/integrations.md:111]:
```suggestion
<br />
```

File: docs/integrations.md (hunk 2 above)
Comment (nbrahms) [reply docs/integrations.md:72]: Does a line break do anything before a section header?

Comment (pabloest) [reply docs/integrations.md:72]: It added a space on my local checkout of the repo.

File: docs/integrations.md [hunk 4]
```diff
@@ -96,7 +69,46 @@ semgrep:
...
 </details>
-</br>
+
+## Standalone providers
+
+Semgrep can also be manually integrated into CI pipelines using a fixed
+ruleset; however features of Semgrep Community like policies and notifications
+are unavailable.
+
+These can be configured for CircleCI or Buildkite as follows:
```
Comment (dlukeomalley) [inline docs/integrations.md:79]: As this list grows and shrinks I think we can remove this line. Thoughts?

File: docs/integrations.md [hunk 5]
```diff
@@ -96,7 +69,46 @@ semgrep:
...
+
+<details><summary>Buildkite</summary>
```
Comment (dlukeomalley) [inline docs/integrations.md:81]: Can we note that deduplication will not work here? From that perspective a `Contact us` might be helpful.

File: docs/integrations.md (hunk 2 above)
Comment (pabloest) [reply docs/integrations.md:72]: Note the “Standalone providers” section:

Without line break:
Screen Shot 2020-11-04 at 2 46 43 PM

---

With line break:
Screen Shot 2020-11-04 at 2 47 19 PM

**Comment by dlukeomalley** [comment]
@nbrahms @pabloest Closing in favor of #39. I've asked @chmccreery to pull over relevant content and changes to that PR.

**Pull request closed without merging**